sdk/storage/Azure.ResourceManager.Storage/tests/Tests/BlobServiceTests.cs

```diff
@@ -38,7 +38,7 @@ public async Task CleanupResourceGroup()
 
         // create container
         // delete container
-        [Test]
+       [RecordedTest]
```

Comment:
What exactly does the annotation do?

Reply:
For tests decorated with this attribute, if the test fails in Playback mode it will automatically attempt to re-run the test in Record mode. If the test successfully re-records the test, it will show as failed, but with the following status message: "Test failed playback, but was successfully re-recorded (it should pass if re-run)."

Reply:
Is there a way to disable this behavior?  Most the time when a test is failing, a manual change is required before it will pass again, so the attempt to re-record will fail almost every time.